packages/docs-examples/components/select/select-virtual-scroll/select-virtual-scroll-example.ts

```diff
@@ -192,4 +201,6 @@ export class SelectVirtualScrollExample {
         this.cdkVirtualScrollViewport.setRenderedContentOffset(0);
         this.cdkVirtualScrollViewport.setRenderedRange(this.initialRange);
     }
+
+    protected readonly OPTIONS = OPTIONS;
```

Comment:
опшены для виртуал скрола можно сгенерить на месте, пример станет проще 
```suggestion
    readonly options = items = Array.from({length: 100000}).map((_, i) => `Option #${i}`);
```